pages/_app.tsx

```diff
@@ -1,12 +1,15 @@
 import { ChakraProvider } from '@chakra-ui/react'
+import '@fontsource/inter/400.css'
+import '@fontsource/inter/600.css'
+import '@fontsource/inter/700.css'
```

Comment:
Adding these imports [was discussed in another PR](https://github.com/MutualAidNYC/services-lists/pull/65#issuecomment-1526862861) but seems like with the Next.js bump, seems like they're necessary for our fonts to show up.